app/src/main/res/layout/activity_main.xml

```diff
@@ -16,191 +16,196 @@
-
+                android:inputType="number"
+                tools:ignore="Autofill,LabelFor" />
         </TableRow>
 
         <TableRow
             android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            >
+            android:layout_height="wrap_content">
 
             <Button
                 android:id="@+id/btnCE"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:text="@string/ce"
+                android:onClick="onCEClick"
                 tools:ignore="ButtonStyle" />
 
             <Button
                 android:id="@+id/btnC"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:text="@string/c"
+                android:onClick="onCClick"
                 tools:ignore="ButtonStyle" />
 
             <Button
                 android:id="@+id/btnDelete"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:text="@string/Delete"
+                android:onClick="onDelClick"
                 tools:ignore="ButtonStyle" />
-
-
         </TableRow>
+
         <TableRow
             android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            >
+            android:layout_height="wrap_content">
 
             <Button
                 android:id="@+id/btnNumber7"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:text="@string/_7"
+                android:onClick="onDigitClick"
                 tools:ignore="ButtonStyle" />
 
             <Button
                 android:id="@+id/btnNumber8"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:text="@string/_8"
+                android:onClick="onDigitClick"
                 tools:ignore="ButtonStyle" />
 
             <Button
                 android:id="@+id/btnNumber9"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:text="@string/_9"
+                android:onClick="onDigitClick"
                 tools:ignore="ButtonStyle" />
 
             <Button
                 android:id="@+id/btnNumberAdd"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:text="@string/plus"
+                android:onClick="onAddClick"
                 tools:ignore="ButtonStyle" />
-
         </TableRow>
 
         <TableRow
             android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            >
+            android:layout_height="wrap_content">
 
             <Button
                 android:id="@+id/btnNumber4"
```

Comment:
you removed code which uses these ids, lets remove id also. Because all +id means that in R class will be +1 field